lib/browser/register-element.ts

```diff
@@ -26,7 +26,7 @@ function patchCallbacks(target: any, targetName: string, method: string, callbac
           if (descriptor && descriptor.value) {
             descriptor.value = wrapWithCurrentZone(descriptor.value, source);
             _redefineProperty(opts.prototype, callback, descriptor);
-          } else {
+          } else if (prototype[callback]) {
```

Comment:
There is already a check for `prototype.hasOwnProperty(callback)` a few lines above.
Does it mean that the properties exists, but are falsy? (That would be unexpected.)

Reply:
Yes, hasOwnProperty return true, but prototype[call back] is undefined.